src/library-view-qt/library-view-qt.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include <QBoxLayout>
+#include <QLineEdit>
+#include <QPointer>
+#include <QWidget>
+
+#include <libaudcore/i18n.h>
+#include <libaudcore/playlist.h>
+#include <libaudcore/plugin.h>
+#include <libaudcore/preferences.h>
+
+#define CFG_ID "library-view"
+
+constexpr auto DEFAULT_LIBRARY_PATH = "~";
+constexpr auto DEFAULT_PLAYLIST_TITLE = "Library view";
```

Comment:
The search-tool plugin was expanded into being a full media library already, as noted in my general comments.  Please do not call this "library view".